File: app/src/main/java/com/werockstar/reactiveandroid/model/GithubUserResponse.java
package com.werockstar.reactiveandroid.model;

import com.google.gson.annotations.SerializedName;

public class GithubUserResponse {

    @SerializedName("name")
    private String name;
    @SerializedName("avatar_url")
    private String avatarUrl;
    @SerializedName("blog")
    private String blog;

    public GithubUserResponse(String name, String avatarUrl, String blog) {
        this.name = name;
        this.avatarUrl = avatarUrl;
        this.blog = blog;
    }

    public String getName() {
        return name;
    }

    public String getAvatarUrl() {
        return avatarUrl;
    }

    public String getBlog() {
        return blog;
    }
}

File: app/src/main/java/com/werockstar/reactiveandroid/model/GithubUserResponse.kt
package com.werockstar.reactiveandroid.model

import com.google.gson.annotations.SerializedName

class GithubUserResponse(@SerializedName("name")
                         val name: String, @SerializedName("avatar_url")
                         val avatarUrl: String, @SerializedName("blog")
                         val blog: String)
